.github/workflows/main.yml

```diff
@@ -20,6 +20,8 @@ jobs:
       traces-api-facade: ${{ steps.filter.outputs.traces-api-facade }}
       transaction-exclusion-api: ${{ steps.filter.outputs.transaction-exclusion-api }}
       finalized-tag-updater: ${{ steps.filter.outputs.finalized-tag-updater }}
+      # If 'false' => all changed files are *.md - https://stackoverflow.com/a/77068717
+      not-markdown-only: ${{ steps.filter-out.outputs.not-markdown-only }}
```

Comment:
I should have scrolled down - `has-code-changes` works for me. Skipping whole doc folders is good too.

Reply:
Good points

I'm thinking `has-changes-requiring-tests`

I will add the image file extensions to the list
- .svg
- .png
- .jpeg
- .jpg
- .gif
- .webp

I don't think we can add an entire folder to the list because `predicate-quantifier: 'every'` means that every filter has to be matched (and not just one which is the default for dorny/paths-filter@v3`

Reply:
I like 
`has-changes-requiring-build`

why: because if we put .jpg/.txt as static asset that shall be bundled, that should be build and published.
Example: UI application may have static assets in it.

Reply:
Good point re: static assets for building hmm...will need to combine a 'file extension blacklist' + 'folder blacklist' then

Actually nvm putting it that way, take back what I said re: not being able to add entire folder to the list